python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py

```diff
@@ -566,9 +567,10 @@ def __init__(
             connection_to_freshness_policy_fn=connection_to_freshness_policy_fn,
         )
         self._workspace_id = workspace_id
-        self._airbyte_resource_def = airbyte_resource_def
-        self._airbyte_instance: AirbyteResource = airbyte_resource_def(
-            build_init_resource_context()
+        self._airbyte_instance: AirbyteResource = (
```

Comment:
change needed because we were relying on `AirbyteResource` being a `ResourceDefinition` (being able to invoke it like a fn)